src/templates/paint_on_template_tool.cpp

```diff
@@ -50,6 +52,23 @@
 
 namespace OpenOrienteering {
 
+namespace {
+
+/// Draws a black or white checkmark, with good contrast to the background color. 
+void drawCheckmark(QPixmap& pixmap, const QColor& background)
+{
+	auto const icon_size = pixmap.width();
+	auto const luminance = 0.3 * background.redF() + 0.59 * background.greenF() + 0.11 * background.blueF();
```

Comment:
Why does this method use another way of luminance computation than `MapperProxyStyle::drawPrimitive()`? FTR, `MapperProxyStyle::drawPrimitive()` uses Qt's `qGray()` function with weights [0.34375; 0.5; 0.15625].

Reply:
`qGray()` is good enough. I just skipped it to quickly when it didn't gave the right color over green.